doc/classes/Plane.xml

```diff
@@ -155,11 +155,11 @@
 		</member>
 	</members>
 	<constants>
-		<constant name="PLANE_YZ" value="Plane( 1, 0, 0, 0 )">
+		<constant name="PLANE_XY" value="Plane( 0, 0, 1, 0 )">
```

Comment:
That's also a `doctool` bug, it shouldn't reorder those, they are registered as YZ, XZ and XY in `core/variant_call.cpp`.